activate.bash

```diff
@@ -1,33 +1,73 @@
 ### (Don't edit below, unless you are a maintener of this package)
 
 #https://unix.stackexchange.com/a/153061
-SOURCED_NAME="$_"
+VIRTUALIZE_SOURCED_NAME="$_"
 
 if [ "${BASH_SOURCE-}" = "$0" ]; then
     echo "You must source this script: \$ source $0" >&2
     exit 33
 fi
 
-SCRIPT_DIR=$( cd -- "$( dirname -- "${SOURCED_NAME}" )" &> /dev/null && pwd )
+if [[ $VIRTUALIZE_ROOT && ! $VIRTUALIZE_ACTIVATING ]]; then
+    echo "virtualizer is active, not activating" $VIRTUALIZE_SOURCED_NAME
+    unset VIRTUALIZE_SOURCED_NAME
+    return
+fi
+
+VIRTUALIZE_NODE_DIR=$( cd -- "$( dirname -- "${VIRTUALIZE_SOURCED_NAME}" )" &> /dev/null && pwd )
 
 # Specify where node should be installed (default is 'node' folder -- probably keep it that way unless you have a good reason)
-LOCAL_NODE_INSTALL_DIRECTORY="$SCRIPT_DIR/node"
-LOCAL_PROJECT_DIRECTORY=$( dirname -- "${SCRIPT_DIR}" )
+VIRTUALIZE_NODE_INSTALL_DIR="$VIRTUALIZE_NODE_DIR/node"
+#VIRTUALIZE_ROOT=$( dirname -- "${VIRTUALIZE_NODE_DIR}" )
```

Comment:
Can drop -- was doing some experimenting with `local`

Reply:
Will instead check around `VIRTUALIZE_NODE_DIR` to check whether or not this has already been activated